prowler/providers/aws/services/ec2/ec2_securitygroup_allow_ingress_from_internet_to_any_port/ec2_securitygroup_allow_ingress_from_internet_to_any_port.py

```diff
@@ -22,14 +25,22 @@ def execute(self):
                 report.resource_id = security_group.id
                 report.resource_arn = security_group.arn
                 report.resource_tags = security_group.tags
-                if not security_group.public_ports:
+                # only proceed if check "..._to_all_ports" did not run or did not FAIL to avoid to report open ports twice
+                if not ec2_client.is_failed_check(
+                    ec2_securitygroup_allow_ingress_from_internet_to_all_ports.__name__,
+                    security_group.id,
+                    security_group.region,
```

Comment:
> What if we use just the ARN?

I thought the same but when you suggested to use `resource_id` and `region` I thought you have valid reasons to do so, so I followed your advice. The ARN should be well enough since it fully qualifies the resource. It hat all the infos, and even more, compared to just `resource_id` and `region`.
So do you agree that I replace `resource_id` and `region` by the ARN?

Reply:
For sure, I said `region` since I just saw the `resource_id` but makes much more sense to place the `ARN`.